servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -151,7 +151,7 @@ void createReferences() throws NessieNotFoundException {
                             .reference(Tag.of(tagName2, null))
                             .create())
                 .isInstanceOf(NessieNotFoundException.class)
-                .hasMessage("Ref 'unknownSource' does not exist"),
+                .hasMessageContainingAll("'unknownSource'", "not"),
```

Comment:
Unrelated to this particular line, but I could not find any other place to comment on this file :|

This test class does not appear to be specific to JAX-RS, but uses our java client for executing test cases... Would it make sense to rename / move test classes to a more generic module for the sake of clarity?

As for me, it was kind of confusing to see the tests from the JAX-RS module reused for Quarkus testing, which is why I assumed JAX-RS was included in Quarkus runtime (while it is not).

I think the same logic applies to `AbstractResteasyTest`, although that one uses plain HTTP requests.

This can be done in a follow-up PR, of course, if you agree.

Reply:
Follow-ups are okay for me.

Reply:
#2119